From 885b90969bf5428452aefa6719ee476a31602e87 Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Tue, 9 May 2023 23:36:21 -0700
Subject: [PATCH 6/7] imx8mn_var_som: use common spl_mmc_get_uboot_raw_sector()

This function moved as part of 8mm to mach-imx/spl.c

Signed-off-by: Michael Scott <mike@foundries.io>
---
 board/variscite/imx8mn_var_som/spl.c | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/board/variscite/imx8mn_var_som/spl.c b/board/variscite/imx8mn_var_som/spl.c
index 54d6d848bdd..f443b5245d3 100644
--- a/board/variscite/imx8mn_var_som/spl.c
+++ b/board/variscite/imx8mn_var_som/spl.c
@@ -278,19 +278,3 @@ void board_init_f(ulong dummy)
 
 	board_init_r(NULL, 0);
 }
-
-#ifdef CONFIG_SPL_MMC
-
-#define UBOOT_RAW_SECTOR_OFFSET 0x40
-unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc)
-{
-	u32 boot_dev = spl_boot_device();
-	switch (boot_dev) {
-		case BOOT_DEVICE_MMC1:
-			return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR;
-		case BOOT_DEVICE_MMC2:
-			return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR - UBOOT_RAW_SECTOR_OFFSET;
-	}
-	return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR;
-}
-#endif
-- 
2.40.1

